packages/waku/src/lib/config.ts

```diff
@@ -51,3 +51,15 @@ export async function resolveConfig() {
   };
   return resolvedConfig;
 }
+
+export const viteInlineConfig = () => {
+  if (process.env.CONFIG_FILE) {
+    return { configFile: path.resolve(process.env.CONFIG_FILE) };
+  }
+  for (const file of ['vite.config.ts', 'vite.config.js']) {
+    if (fs.existsSync(file)) {
+      return { configFile: path.resolve(file) };
+    }
+  }
+  return {};
+};
```

Comment:
I'm not super comfortable with fs.existsSync, but this is what I came up with.

Reply:
what's the problem of vite resolveConfig? Can't we use that?

Reply:
I tried that, but it didn't work very well, because we have two `root` configs. There might be my oversight though.
Maybe we should re-consider changing `root`. Let's try that after merging this PR.

Reply:
Yes, I tried the root on where the vite.config is and it worked well! Let me know your findings, because having multiple roots makes loading configs really hard (and trying to keep them consistent would be so hard in the future)

Reply:
Yeah, we want `src` and `dist` at the same level.
Can you open a PR that only changes it? (=not changing "root")

Reply:
Yes, I can work on it tomorrow if you're ok with it. Or later tonight.

Reply:
Yes, please go ahead. But, note that it may not work as we want. So, I mean a draft PR to show your idea. We can then consider to move forward.